1-org/envs/shared/variables.tf

```diff
@@ -95,3 +95,113 @@ variable "log_export_storage_location" {
   type        = string
   default     = "US"
 }
+
+
+variable "dns_hub_project_alert_spent_percents" {
+  description = "A list of percentages of the budget to alert on when threshold is exceeded for the DNS hub project."
+  type        = list(number)
+  default     = [0.5, 0.7, 1.0]
```

Comment:
I think that the default values for the thresholds should be 50%, 75%, 90%, and 95% according to the Whitepaper.

Reply:
done